lib/dl_formula/dl_formula/mutation/lod.py

```diff
@@ -29,21 +26,19 @@ class ExtAggregationToQueryForkMutation(DimensionResolvingMutationBase):
     """
 
     def match_node(self, node: nodes.FormulaItem, parent_stack: Tuple[nodes.FormulaItem, ...]) -> bool:
-        is_agg = is_aggregate_function(node)
-        direct_parent = parent_stack[-1]
-        already_patched = (
```

Comment:
This check used to prevent an infinite loop: when we replace `agg_expr` with `QueryFork(agg_expr)`, we used to traverse children of the new node, but this would result in the same match/replace for the `agg_expr` node. Because we are now traversing nodes from children to parents, it is no longer required.